index.js

```diff
@@ -63,10 +66,49 @@ function toggleColor(element) {
     if (element.style.color === "green") {
         element.style.color = "black"
     } else {
-        element.style.color = "green"
+        element.style.color = "red"
     }
 }
 
+header.addEventListener('click', function() {
+    toggleColor(header)
+})
+
```

Comment:
Nicely done! Using arrow functions (ES6 feature), you could also write this solution as shown below. This is the same exact functionality!
`header.addEventListener("click", () => toggleColor(header))`